README.md

```diff
@@ -150,7 +150,47 @@ target:
 `subpath` in the previous url is optional in case your charts are not stored directly under your projects.
 It is worth mentioning that you can use Harbor robot accounts using OCI registries as source or target.
 
-Also, take into account that if you use OCI as the source repository you must specify the list of chart to synchronize
+Also, take into account that if you use OCI as the source repository you must specify the list of charts to synchronize
+or a pointer to an [index file](#helm-charts-index-for-oci-based-repositories) in the repository.
+
+#### Helm Charts Index for OCI-based repositories
+
+By using a charts index file for OCI-bases repository you won't need to maintain a hardcoded list of chart names in the config file.
+charts-syncer will be able to auto-discover what charts need to be synchronized.
```

Comment:
out of curiosity, if I put the filter in the file but the index is also there what happens? Does it work as a filter?